hermes-common/src/test/java/pl/allegro/tech/hermes/common/message/wrapper/AvroMessageContentWrapperTest.java

```diff
@@ -71,14 +72,42 @@ public void shouldUnwrapAvroMessageAndGenerateMetadataWhenNotExists() throws Thr
         UnwrappedMessageContent unwrappedMessage = avroMessageContentWrapper.unwrapContent(wrappedMessage, avroUser.getCompiledSchema());
 
         //then
-        assertThat(unwrappedMessage.getMessageMetadata().getId()).isNotEmpty();
+        assertThat(unwrappedMessage.getMessageMetadata().getId()).isEmpty();
```

Comment:
While this test is in package common, I'm not sure whether this test makes sense. It would make sense if it was in consumers package - then we'd be sure it's the case of non-generated ID's. But if this code can be used in both consumers and frontend, I'd suggest clearly defining where the wrapping/unwrapping can happen and moving to consumers code if necessary.

Reply:
Ok ```AvroMessageContentWrapper.unwrapContent()``` is package private so we need to use higher level classes to test it but I understand your intentions.

Reply:
I think writing this kind of test in consumers package (with appropriate name will be best option). What do you think ?

Reply:
I had a look at the usage of `AvroMessageContentWrapper` and noticed it's used in consumers and management code. At this point perhaps it would be enough to add a javadoc describing the responsibility of the class - that it does not generate any ID, only deals with wrapping and unwrapping what is provided?